src/main/java/com/fitpet/server/user/presentation/controller/UserController.java

```diff
@@ -75,10 +83,35 @@ public ResponseEntity<UserDto> update(
         return ResponseEntity.status(HttpStatus.OK).body(user);
     }
 
+    @GetMapping("/rankings/daily-step")
+    public ResponseEntity<RankingResponse> getDailyStepRanking() {
+        // TODO: 추후 토큰에서 userId 추출 로직으로 교체
+        Long userId = 3L;
+        log.info("[UserController] 일일 걸음 랭킹 조회 요청: userId={}", userId);
+        RankingResult rankingResult = userService.getDailyStepRanking(userId);
+        RankingResponse rankingResponse = new RankingResponse(
+                rankingResult.top10().stream().map(this::toDto).toList(),
+                rankingResult.myRank()
+        );
```

Comment:
`RankingResponse`와 `UserRankingDto`를 내부에 정적 팩토리 메소드를 구현하여 
변환로직(stream, map)을 DTO 내부로 캡슐화하는 것이 어떨까용??🤔

수정하면 Controller가 데이터 변환 책임에서  벗어나서 훨씬 간결해집니다!

```suggestion
        RankingResult result = userService.getDailyStepRanking(userId);
        
        return ResponseEntity.ok(RankingResponse.from(result));
```

Reply:
옹 새로운 내용 배워갑니다 감사합니다 !!
좋은 의견인거 같아요 👍 
해보니까 훨씬 간결해지고 보기도 좋은 것 같습니다 ㅎ.ㅎ